source/examples/rapids-coiled-cudf/notebook.ipynb

```diff
@@ -0,0 +1,2461 @@
+{
```

Comment:
<p>Maybe add a conclusion section, with links to docs to cudf pandas, and link to the blog when is available. Probably also add links to the deployment docs too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>